### DISCUSSION TITLE
Plex LXC: HW transcode for AMD iGPU?

### DISCUSSION THREAD

**Discussion by jotpunktopunkt** [tteck/Proxmox#2317] [Help Needed]
hi!

i was recommended the tteck plex lxc script to get a plex container running with hw transcode, but i couldn't find an answer wether this would work for AMD iGPUs too (i use a ryzen 3625u)
_i'm not code-savvy, but looking at the script i couldn't tell wether it enables hw transcode at all or if another (additional script is needed)_
edit: i was blind...i now see the code that enables transcoding ... but i don't understand if it would work for amd as well

what are the steps (or guides) needed to enable gpu passthrough for an amd igpu in the tteck plex lxc?

and i have never used LXCs before...can i break something with the script?
can i try the plex lxc and enable gpu passthrough - and simply delete it if it's not working without crashing my PVE node?

sry for the noob-ish questions - i have never passed through hw to a vm or a container, so i try to make baby steps to not break something ;)

**Comment by MilesTEG1** [comment]
Hello,
@jotpunktopunkt 
Have you try the script on a AMD CPU with iGPU? Does it works?

**Comment by 5ulo** [comment]
just stick with this tutorial.. easy, no problems. My Ryzen 5 8500G with iGPU was set properly with hw transcoding.
https://www.derekseaman.com/2023/04/proxmox-plex-lxc-with-alder-lake-transcoding.html
